src/psyclone/tests/psyir/nodes/whileloop_test.py

```diff
@@ -0,0 +1,173 @@
+# -----------------------------------------------------------------------------
```

Comment:
Please re-name to "while_loop_test.py" and copyright date should just be 2023.

Reply:
Done.